scripts/zones/Buburimu_Peninsula/mobs/Wake_Warder_Wanda.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------
+-- Area: Buburimi Peninsula
+--  MOB: Wake Warder Wanda
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_MAGIC_COOL, 50); -- just one spell to spam
+    mob:addMod(MOD_SILENCERES,20);
+    mob:addMod(MOD_BINDRES,20);
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobEngaged
+-----------------------------------
+
+function onMobEngaged(mob,target)
+    mob:setMod(MOD_REGAIN, 25);
```

Comment:
which it is https://github.com/DarkstarProject/darkstar/pull/3218/files#diff-7dc3599402cd88948b941c882c5a7f48R38